packages/python/plotly/_plotly_utils/colors/_swatches.py

```diff
@@ -33,7 +33,7 @@ def _swatches(module_names, module_contents, template=None):
                 marker=dict(color=colors),
                 hovertemplate="%{y}[%{customdata}] = %{marker.color}<extra></extra>",
             )
-            for name, colors in reversed(sequences)
+            for name, colors in (sequences)
```

Comment:
i actually kind of liked the old order, otherwise they read bottom up...

Reply:
oh good catch, I had modified this while debugging and then forgot to change it back.